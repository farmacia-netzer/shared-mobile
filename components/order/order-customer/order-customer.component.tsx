import React from 'react';
import { StyleSheet, View } from 'react-native';

import { TProfile } from '#types';
import { NetzerKeyValueText } from '##component/netzer-key-value-text/netzer-key-value-text.component';

interface OrderCustomerProp {
  profile: TProfile;
}

export const OrderCustomer: React.FC<OrderCustomerProp> = ({ profile: { fullName, cellphone } }: OrderCustomerProp) => {
  return (
    <View style={styles.container}>
      <NetzerKeyValueText header="Nombre" value={fullName} />
      <NetzerKeyValueText header="Celular" value={cellphone} />
    </View>
  );
};

const styles = StyleSheet.create({
  container: {}
});
