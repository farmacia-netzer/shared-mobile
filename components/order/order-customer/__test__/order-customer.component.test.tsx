import React from 'react';
import { render } from '@testing-library/react-native';

import { OrderCustomer } from '../order-customer.component';

describe('<OrderCustomer />', () => {
  it('renders', () => {
    const { toJSON } = render(
      <OrderCustomer
        profile={{
          firstName: '',
          lastName: '',
          fullName: '',
          cellphone: '',
          email: '',
          id: ''
        }}
      />
    );
    expect(toJSON()).toMatchSnapshot();
  });
});
