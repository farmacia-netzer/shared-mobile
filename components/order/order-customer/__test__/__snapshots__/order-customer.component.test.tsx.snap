// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<OrderCustomer /> renders 1`] = `
<View
  style={Object {}}
>
  <View>
    <Text
      fontStyle={12}
      style={
        Array [
          Object {
            "fontSize": 14,
            "lineHeight": 18,
          },
          undefined,
          Object {
            "fontSize": 14,
            "marginBottom": 5,
            "marginTop": 15,
          },
        ]
      }
    >
      Nombre
    </Text>
    <Text
      fontStyle={12}
      style={
        Array [
          Object {
            "fontSize": 14,
            "lineHeight": 18,
          },
          undefined,
          Object {
            "fontSize": 20,
            "fontWeight": "400",
            "lineHeight": 24,
          },
        ]
      }
    />
  </View>
  <View>
    <Text
      fontStyle={12}
      style={
        Array [
          Object {
            "fontSize": 14,
            "lineHeight": 18,
          },
          undefined,
          Object {
            "fontSize": 14,
            "marginBottom": 5,
            "marginTop": 15,
          },
        ]
      }
    >
      Celular
    </Text>
    <Text
      fontStyle={12}
      style={
        Array [
          Object {
            "fontSize": 14,
            "lineHeight": 18,
          },
          undefined,
          Object {
            "fontSize": 20,
            "fontWeight": "400",
            "lineHeight": 24,
          },
        ]
      }
    />
  </View>
</View>
`;
